api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementServiceTest.java

```diff
@@ -165,7 +167,7 @@ void shouldReturnIneligibleForExistingClaimantWithoutVouchers() {
         EligibilityAndEntitlementDecision result =
                 eligibilityAndEntitlementService.evaluateExistingClaimant(claimant, cycleStartDate, previousCycle);
 
-        assertCorrectResult(result, INELIGIBLE, eligibilityResponse, voucherEntitlement);
+        assertCorrectResult(result, INELIGIBLE, CONFIRMED, eligibilityResponse, voucherEntitlement);
```

Comment:
I am surprised not to see one of these tests have the status values of `INELIGIBLE, NOT_CONFIRMED`, is one of these tests wrong, or is there a missing test? Was expecting to see some tests with different `QualifyingBenefitEligibilityStatus` values being asserted.

Reply:
I have added a test for this, but also refactored the DuplicateClaimChecker. I'll put both in a separate pr.